jax/_src/pjit.py

```diff
@@ -1147,7 +1147,7 @@ def unpack(key):
 
   # have we seen this function before at all?
   fun_name = getattr(f, '__qualname__', f)
-  if debug_info.func_src_info:
+  if debug_info and debug_info.func_src_info:
```

Comment:
if debug_info is not None?

Reply:
It cannot be falsy because it's a non-empty namedtuple, but sure.